### ISSUE TITLE
Signups are currently disabled

### ISSUE THREAD

**Issue by honghuoming** [withfig/fig#2832]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my feature

### Description:
> Please include a detailed description of the feature (and an image or screen recording, if applicable)
Signups are currently disabled
<img width="1066" alt="image" src="https://github.com/withfig/fig/assets/43715311/d83ced17-f1cb-4add-b7ad-d124088a530b">

**Comment by tomvonheill** [comment]
Alright so i fixed it because I was trying to login with the wrong username and it got stuck in this location. I just needed to type `fig logout` from the terminal and then relogin

**Comment by rami0101-AC** [comment]
Having the same issue. I got a new computer, and I cannot log in to fig.

**Comment by lukejans** [comment]
> Thinking out loud: Is this related to fig team having joined AWS and is rebranded to Codewhisperer now? (Just tried that variant and its sort of useless since the dashboard has NOTHING in common whaat FIG used to be.
> 
> In case anyone want to waster his/her time on it note that auth will not work on Safari + it enables telemetry and sharing commands with AWS (no idea why I want AWS to log every(?!) command (including secrets, pw, ...?! - I DO NOT KNOW BUT AM NOT WILLING TO RISK IT)

I wouldn't be massively worried about security with amazon as the majority of devs already have put their trust in AWS with EC2. CodeWhisperer / CW will work the same in terms of command line use the only difference is that ai is now free for text to bash translations (thank you amazon, in your face fig for making it a paid feat). I'm assuming you're referring to dotfiles and configuration which yes CodeWhisperer does not have atm. I don't really see a use for this feature as it was very buggy on figs platform and managing my own dotfiles with symlinks and other GNU tools seems to be the better method.

**Comment by seanchiggins** [comment]
Having the same issue. Any updates?

**Comment by lukejans** [comment]
> Having the same issue. Any updates?

yes as i mentioned above fig has been bought out by amazon so as part of their contract they aren't allowed to have anyone signup for fig and you're only able to signup for amazons CodeWhisperer pretty much the same thing.

[CodeWhisperer](https://docs.aws.amazon.com/codewhisperer/latest/userguide/command-line.html) (prev. fig.io)

**Comment by leejk3** [comment]
Having the same issue. :(

**Comment by aausek** [comment]
Same issue here. Tried via Fig app and terminal but still get the 400 error.
